### ISSUE TITLE
Runing in Mixed Mode --opencl-platform option 

### ISSUE THREAD

**Issue by andrewpsp** [ethereum-mining/ethminer#654]
When running in _Mixed mode_ use the **X** flag if I pass the **--opencl-platform** option with 1 the application doesn't execute nor does it provide any error message or output :cry:

* Command run : 
```  
 ethminer -X -SP 1 -S eth-us.dwarfpool.com:8008  
 -O 0xE500eA1e6B27077EBb97757B6817d954c3eb33a6:x \
 --opencl-platform 1
```

**Comment by SnowLeopard71** [comment]
What is the output of --list-devices?    
I have 5 Nvidias and 1 AMD running in mixed-mode and I have to give it 0 as the parameter to --opencl-platform.   
(Works great except temp & fan speed are not reported for the AMD card.)

**Comment by DeadManWalkingTO** [comment]
Try new version and feedback.
Thank you!

**Comment by AndreaLanfranchi** [comment]
Resolved by #1704
